deploy/helm/values.yaml

```diff
@@ -16,4 +16,4 @@ configMap:
   cachingCpuRequest: ".05"
   cachingCpuLimit: ".2"
   nodeSelector: "{}"
-
+  imagePullSecrets: ""
```

Comment:
No new line in the end

Reply:
There actually is a new line in the end here ...

Reply:
yup, looks good